src/base/logger.h

```diff
@@ -82,8 +82,13 @@ class Logger : public QObject
 
     void addMessage(const QString &message, const Log::MsgType &type = Log::NORMAL);
     void addPeer(const QString &ip, bool blocked, const QString &reason = {});
+    Log::Msg message(int index) const;
+    Log::Peer peer(int index) const;
     QVector<Log::Msg> getMessages(int lastKnownId = -1) const;
```

Comment:
maybe `getMessagedFrom::(int lastKnownId)`?

Reply:
>maybe getMessagedFrom(int lastKnownId)?

No. As I said before this name is meaningless from Logger perspective (since it knows all its messages). It can be just `getMessages(int startId)` which literally means "give me all the messages starting with some specific ID".